src/core/public/injected_metadata/injected_metadata_service.ts

```diff
@@ -80,6 +80,9 @@ export interface InjectedMetadataParams {
         user?: Record<string, UserProvidedValues>;
       };
     };
+    apm: {
+      [key: string]: unknown;
+    };
```

Comment:
Quick note on this: injectedMetadata are more or less already deprecated and might be removed at some point in NP. You have to keep in mind that you need to migrate this to an asynchronous way to access and use this data at some point.